user/languages/julia.md

```diff
@@ -37,6 +37,7 @@ julia:
 Acceptable formats are:
  - `nightly` will test against the latest [nightly build](https://julialang.org/downloads/nightlies.html)
 of Julia.
+ - `X` will test against the latest release for that major version.
```

Comment:
Technically only true for 1.x releases; just specifying "0" won't do anything, for example.
```suggestion
 - `X` will test against the latest release for that major version. (Applies only to major versions 1 and later.)
```